yoti-sdk-parent/pom.xml

```diff
@@ -79,22 +79,24 @@
     <maven.compiler.target>${supported.java.version}</maven.compiler.target>
     
     <!-- sdk dependency versions -->
-    <slf4j.version>1.7.36</slf4j.version>
+    <slf4j.version>2.0.6</slf4j.version>
```

Comment:
FYI: This makes Java 8 the min required version for the JDK. I'm fine with it but just pointing this out in case supporting Java 7 is needed (if so it needs to be 1.7.36)

Reply:
thanks for point it out. Since v3.0.0 the minimum required is Java 8 so we're good with this change